cfgov/unprocessed/css/pages/ask.less

```diff
@@ -3,44 +3,41 @@
    Ask CFPB pages at /ask-cfpb/.
    ========================================================================== */
 
+/* Answer page */
 
-.answer-module {
-    border-top: 3px solid @green;
-    border-bottom: 1px solid @block__border-bottom;
-    padding: unit( ( @grid_gutter-width / 2 ) / @base-font-size-px, em ) 0;
-    margin: unit( @grid_gutter-width / @base-font-size-px, em ) 0;
-    color: @gray;
-
-    .respond-to-min(@bp-sm-min, {
-        padding: unit( ( @grid_gutter-width / 2 ) / @base-font-size-px, em );
-    });
-
-    .respond-to-min(@bp-med-min, {
-       width: 80%;
-    });
-
-    h1, h2, h3, h4, h6 {
-        .heading-5();
-        margin-top: 0;
-        margin-bottom: unit( ( @grid_gutter-width / 3 ) / @base-font-size-px, em );
-    }
+.ask-answer-page {
+    .answer-module {
+        border-top: 3px solid @green;
+        border-bottom: 1px solid @block__border-bottom;
+        padding: unit( ( @grid_gutter-width / 2 ) / @base-font-size-px, em ) 0;
+        margin: unit( @grid_gutter-width / @base-font-size-px, em ) 0;
+        color: @gray;
+
+        .respond-to-min(@bp-sm-min, {
+            padding: unit( ( @grid_gutter-width / 2 ) / @base-font-size-px, em );
+        });
+
+        .respond-to-min(@bp-med-min, {
+           width: 80%;
```

Comment:
On larger screens, the tips/warnings become slightly narrower so they stand out from the surrounding text:

<img width="777" alt="screen shot 2017-05-15 at 1 53 37 pm" src="https://cloud.githubusercontent.com/assets/778171/26079005/2617abb0-3976-11e7-97c7-a2084a033da2.png">